base/cvd/cuttlefish/host/commands/cvd/parser/instance/cf_boot_configs.cpp

```diff
@@ -18,33 +18,45 @@
 #include <string>
 #include <vector>
 
-#include "json/json.h"
-
+#include "common/libs/utils/base64.h"
 #include "common/libs/utils/result.h"
+#include "cuttlefish/host/commands/cvd/parser/load_config.pb.h"
 #include "host/commands/assemble_cvd/flags_defaults.h"
 #include "host/commands/cvd/parser/cf_configs_common.h"
 
 namespace cuttlefish {
 
-Result<void> InitBootConfigs(Json::Value& instances) {
-  for (auto& instance : instances) {
-    CF_EXPECT(InitConfig(instance, CF_DEFAULTS_ENABLE_BOOTANIMATION,
-                         {"boot", "enable_bootanimation"}));
-    CF_EXPECT(InitConfig(instance, CF_DEFAULTS_EXTRA_BOOTCONFIG_ARGS,
-                         {"boot", "extra_bootconfig_args"}));
+using cvd::config::EnvironmentSpecification;
+using cvd::config::Instance;
+
+static bool EnableBootAnimation(const Instance& instance) {
+  const auto& boot = instance.boot();
+  if (boot.has_enable_bootanimation()) {
+    return instance.boot().enable_bootanimation();
+  } else {
+    return CF_DEFAULTS_ENABLE_BOOTANIMATION;
+  }
+}
+
+static Result<std::string> BtCfg(const Instance& instance) {
```

Comment:
nit: `BtCfg` made think of bluetooth instead of boot. Rename as `BootCfg` or just `Cfg`

Reply:
Done.